tcms/testruns/templates/testruns/search.html

```diff
@@ -114,11 +205,15 @@
 </div><!-- /container -->
 
 <!-- JavaScript that is used in this page -->
-<script src="{% static 'bootstrap-switch/dist/js/bootstrap-switch.min.js' %}"></script>
-<script src="{% static 'bootstrap-select/dist/js/bootstrap-select.min.js' %}"></script>
+    <script src="{% static 'moment/min/moment-with-locales.min.js' %}"></script>
+    <script src="{% static 'eonasdan-bootstrap-datetimepicker/build/js/bootstrap-datetimepicker.min.js' %}"></script>
+    <script src="{% static 'bootstrap-switch/dist/js/bootstrap-switch.min.js' %}"></script>
+    <script src="{% static 'bootstrap-select/dist/js/bootstrap-select.min.js' %}"></script>
+
+    <script src="{% static 'js/jsonrpc.js' %}"></script>
+    <script src="{% static 'js/utils.js' %}"></script>
+    <script src="{% static 'js/pagination.js' %}"></script>
+    <script src="{% static 'testruns/js/search.js' %}"></script>
+
```

Comment:
Please don't introduce unrelated changes, especially when changing the order of the lines. This makes it very hard to figure out if something essential has been changed or not. And if it has been then answering the question: is there something missing or everything looks fine. 

If nothing is missing and all looks fine then there's very little reason for such changes in the first place other than eye candy. If such is the case then formatting changes need to be introduced separately of functional related changes because of the above mentioned reasons.